cmd/jimmsrv/service/service.go

```diff
@@ -194,14 +194,15 @@ type Params struct {
 
 // A Service is the implementation of a JIMM server.
 type Service struct {
-	jimm jimm.JIMM
+	jimmParameters jimm.Parameters
```

Comment:
because we have many methods that just create "stuff" (e.g. oauth authenticator) needed by jimm. and all that "stuff" gets put into the parameters and then passed on to the constructor